core/domain/suggestion_services_test.py

```diff
@@ -1765,17 +1765,31 @@ def test_create_raises_for_suggestion_type_not_on_contributor_dashboard(
             self):
         edit_state_content_suggestion = (
             self._create_edit_state_content_suggestion())
+        # Mocking the SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS dict in
+        # suggestion services so that this test still passes if the
+        # "edit state content" suggestion type is added to the Contributor
+        # Dashboard in the future.
+        suggestion_emphasized_text_getter_functions_mock = {
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT: (
+                lambda suggestion: suggestion.change.translation_html),
+            suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: (
+                lambda suggestion: suggestion.change.question_dict[
+                    'question_state_data']['content']['html'])
+        }
```

Comment:
You could probably just use an empty dict for the purposes of this test (e.g. no type is valid).

Reply:
Okay, I made it an empty dict - thanks!